apps/concept-catalog/app/page.tsx

```diff
@@ -1,6 +1,9 @@
+import { cookies } from 'next/headers';
 import { redirect } from 'next/navigation';
 
 const Home = async () => {
```

Comment:
https://nextjs.org/docs/app/api-reference/file-conventions/page#searchparams-optional

Bruke `searchParams`?

Reply:
Blir mer kode og gjør akkurat det samme. Da må man virkelig bruke searchParams for at det skal fungere. Akkurat her bruker vi ikke noe som ligger i searchParams.